src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -105,9 +105,14 @@ def __init__(self, config):
 
         # Keep a record of the thresholds in memory
         self.currentRcThresholds = {}
-
-        self.reqmgr2Svc = ReqMgr(self.config.TaskArchiver.ReqMgr2ServiceURL)
-        self.abortedAndForceCompleteWorkflowCache = self.reqmgr2Svc.getAbortedAndForceCompleteRequestsFromMemoryCache()
+        
+        self.useReqMgrForCompletionCheck = getattr(self.config.TaskArchiver, 'useReqMgrForCompletionCheck', True)
+        
+        if self.useReqMgrForCompletionCheck:
+            # only set up this when reqmgr is used (not Tier0)
+            self.reqmgr2Svc = ReqMgr(self.config.TaskArchiver.ReqMgr2ServiceURL)
+            self.abortedAndForceCompleteWorkflowCache = self.reqmgr2Svc.getAbortedAndForceCompleteRequestsFromMemoryCache()
```

Comment:
We can't do this 
 self.abortedAndForceCompleteWorkflowCache is not list it has method to .getData(), although self.abortedAndForceCompleteWorkflowCache will be never used in else case.